travis/build.sh

```diff
@@ -3,11 +3,12 @@ set -ex
 # show available schemes
 # xcodebuild -list -project ./BridgeSDK.xcodeproj
 if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-    bundle exec fastlane test scheme:"BridgeSDK"
-elif [ "$TRAVIS_BRANCH" = "master" ]; then
-    bundle exec fastlane test scheme:"BridgeSDK"
+    bundle exec fastlane doc scheme:"BridgeSDK"
 fi
-if [ ! -z "$TRAVIS_TAG" ]; then
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
     bundle exec fastlane doc scheme:"BridgeSDK"
+    openssl aes-256-cbc -K $encrypted_d59c41fdd72b_key -iv $encrypted_d59c41fdd72b_iv -in BridgeAdminCredentials.plist.enc -out BridgeAdminCredentials.plist -d
```

Comment:
We've decided setup ENV vars for secrets instead of committing encrypted files to the repo.